examples/udpmulticat.c

```diff
@@ -1,270 +1,189 @@
-/*
```

Comment:
Can you create a new example instead of modifying udpmulticat ?

Reply:
Like ARQ, I'm not sure if we should do a proper example, this is just for quick testing at the moment

Reply:
IMO it's always interesting to have an example implementation of the API.